tic_tac_toe/game.py

```diff
@@ -63,22 +87,42 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    #Check if player has all three along a horizonal or vertical:
+    for coordinate in range(0,3):
+        if _is_winning_combination(board, ((coordinate,0), (coordinate,1), (coordinate,2)), player) \
+            or _is_winning_combination(board, ((0,coordinate), (1,coordinate), (2,coordinate)), player):
+                return player
+
+    # Check if a player has either diagonal:
+    if _is_winning_combination(board, ((0,0), (1,1), (2,2)), player) \
+        or _is_winning_combination(board, ((0,2), (1,1), (2,0)), player):
+            return player
+
+    return None
 
 
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
+    #board = [['-']*3]*3  # <---- Does not work???
+    board = [['-','-','-'],['-','-','-'],['-','-','-']]
+    defaultgame = {
+        'player1': player1,
+        'player2': player2,
+        'board': board,
+        'next_turn': 'X',
```

Comment:
Technically, the program gets passed a value for player1 and player2 when the class in test_main.py is run, so if the person running the code so wanted to change player1 = 'L' and player2 = 'Y', you would have a bug.